roster2/tool/src/webapp/js/card-game/card-game.js

```diff
@@ -184,7 +184,7 @@ export default class CardGame extends BaseGame {
         const allUsersCount = this.state.allUsers.length;
         const learnedUsersCount = allUsersCount - this.state.learnUsers.length;
 
-        const imageSrc = this.config.officialImage
+        const imageSrc = this.config.showOfficialPhoto
```

Comment:
@stetsche Could you check this? Not in a hurry, enjoy your holidays first!

Reply:
Looks correct!